### DISCUSSION TITLE
Google calendar not deisplaying appointments added in Thunderbird calendar.

### DISCUSSION THREAD

**Discussion by rgarnett1955** [kewisch/gdata-provider#794] [Support]
Hi,

I am trying to get thunderbird to maintain a calendar that I can access from another any of my PC's which use Thunderbird.

I installed the Google provider, but I am not seeing my thunderbird calendar entries (appointments) in the Google Calendar.

Is it supposed to do this?

I can make an entry in the Google calendar and it appears in Thunderbird, but not the other way around.

Kind Regards
Rob

**Comment by DDzwiedziu** [comment]
Same, I even made a bug report to TB by mistake (with logs): https://bugzilla.mozilla.org/show_bug.cgi?id=1919016

**Comment by paniash** [comment]
Any updates on this? I also see this problem.
